migration/ccd-to-am-migration-runner.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+echo
+set -e
+set -u
+
+DB="am"
+USER="amuser"
```

Comment:
DB hostname/IP is also required to connect to a remote db

Reply:
👍